PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/tasksmanager/MainThreadExecutor.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *    Copyright 2020 APPNEXUS INC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package org.prebid.mobile.tasksmanager;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.VisibleForTesting;
+
+public class MainThreadExecutor implements CancellableExecutor {
+    private final Handler mHandler = new Handler(Looper.getMainLooper());
```

Comment:
can we use `handler` name instead of `mHandler`. Generally we don't use member prefix